store/pg_engine.go

```diff
@@ -116,6 +116,24 @@ func (db *DB) Open(ctx context.Context) (err error) {
 	if err := d.SetupMigrationIfNeeded(ctx); err != nil {
 		return err
 	}
+	// TODO(d): remove this block once all existing customers all migrated to semantic versioning.
+	if _, err := getLatestVersion(ctx, d, databaseName); err != nil {
+		// Convert existing record to semantic versioning format.
+		if !strings.Contains(err.Error(), "invalid stored version") {
+			return err
+		}
+		db, err := d.GetDbConnection(ctx, "bytebase")
+		if err != nil {
+			return err
+		}
+		if _, err := db.ExecContext(ctx, "UPDATE migration_history SET version = '0001.0000.0000-20210113000000' WHERE id = 1 AND version = '10000';"); err != nil {
+			return err
+		}
+		if _, err := db.ExecContext(ctx, "UPDATE migration_history SET version = '0001.0000.0000-20210113000001' WHERE id = 2 AND version = '10001';"); err != nil {
```

Comment:
I thought we only have version 10000 in the prod.  10001 is only in non-prod. If that's the case, I would not worry about changing 10001.

The small side-benefit is we can explicitly comment why we need to update 10000 because that's the only version in the prod.

Reply:
10000 was for CREATE DATABASE and 10001 was for creating the schema.

Reply:
I see. Let's add the comment then, pointing to https://github.com/bytebase/bytebase/blob/release/v1.0.2/store/pg_engine.go#L251 and https://github.com/bytebase/bytebase/blob/release/v1.0.2/store/pg_engine.go#L295

Reply:
Done.